### DISCUSSION TITLE
Can't sync or update Lazyvim

### DISCUSSION THREAD

**Discussion by deepseeksss** [LazyVim/LazyVim#3634] [Q&A] [answered]
Failed (2)
    ○ persistence.nvim  BufReadPre  <leader>ql  <leader>qd  <leader>qs 
        error: cannot update the ref 'refs/remotes/origin/main': unable to append to '.git/logs/refs/remotes/origin/main': Permission denied
        From https://github.com/folke/persistence.nvim
         ! 5fe0770..95d03ad  main       -> origin/main  (unable to update local ref)
        error: cannot update the ref 'refs/remotes/origin/release-please--branches--main--components--persistence.nvim': unable to append to '.git/logs/refs/remotes/origin/release-please--branches--main--components--persistence.nvim': Permission denied
         ! e9f1189...55b0b4f release-please--branches--main--components--persistence.nvim -> origin/release-please--branches--main--components--persistence.nvim  (unable to update local ref)
    ● trouble.nvim 3.18ms 󰢱 trouble  lualine.nvim
        error: cannot update the ref 'refs/remotes/origin/release-please--branches--main--components--trouble.nvim': unable to append to '.git/logs/refs/remotes/origin/release-please--branches--main--components--trouble.nvim': Permission denied
        From https://github.com/folke/trouble.nvim
         ! cf5f30d...eca0829 release-please--branches--main--components--trouble.nvim -> origin/release-please--branches--main--components--trouble.nvim  (unable to update local ref

**Comment by rubiin** [comment, marked as answer]
Maybe a permission issue , if you are on unix based system
https://stackoverflow.com/questions/2642836/git-error-unable-to-append-to-git-logs-refs-remotes-origin-master-permission

If you have hosted your config on github or somewhere else, you can try to do delete the local config and reinstall as a final resort
